docs/development/development-environment-docker/README.md

```diff
@@ -350,6 +355,8 @@ security.pki.certificateFiles = [ path_to_root_ca.crt ];
 ```
 
 Then rebuild your system. After that the generated root CA should be inside `/etc/ssl/certs/ca-certificates.crt`.
+However, you cannot mount this directly into the containers, since it's just a link. Use `readlink -f` to find out where
+the link points in your nix store, and mount that file directly.
```

Comment:
@ba1ash Didn't you mount the bundle directly? Why does the volume mount is different for you and @b12f ?

Reply:
I tried to mount the link directly and it works for me. So, I switched to this approach.

Reply:
Really? unless you mounted the whole nix store the link should be pointing at `/nix/store/...` which doesn't exist inside the container. I ran into this exact issue when trying it myself. Does `docker compose exec backend bash -c "cat /etc/ssl/certs/ca-bundle.crt | grep Open"` produce output?

Reply:
Yes, it works for me..
![image](https://github.com/opf/openproject/assets/11717656/7ce78970-8829-44af-9ae9-a047e70fd101)

Reply:
this might be important to check it first

Reply:
Hmm I tried now and you're right it works. Maybe this is new docker behavior?

Reply:
> Hmm I tried now and you're right it works. Maybe this is new docker behavior?

Maybe. I did not manage to find out whether it is new or old behaviour.